app/views/admin/base/_policy_area_fields.html.erb

```diff
@@ -1,6 +1,6 @@
 <% cache_if edition.topic_ids.empty?, taggable_topics_cache_digest do %>
   <fieldset class="edition-topic-fields">
-    <%= form.label :topic_ids, 'Policy Areas', required: edition.must_be_tagged_to_policy_area? %>
```

Comment:
Haha, this will be the source of the infamous red *

Reply:
Yes! I forgot I removed it in this story.